app/assets/stylesheets/application.css.scss

```diff
@@ -136,7 +183,7 @@ h5 {
         padding-left: 5rem; }
     .footer .contact i {
       position: absolute;
-      background-color: #33383b;
+      background-color: #461B7E;
```

Comment:
Color literals like `#461B7E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#461B7E` should be written as `#461b7e`